src/argilla/server/alembic/versions/7cbcccf8b57a_create_metadata_properties_table.py

```diff
@@ -33,12 +33,14 @@ def upgrade() -> None:
     # ### commands auto generated by Alembic - please adjust! ###
     op.create_table(
         "metadata_properties",
+        sa.Column("id", sa.Uuid(), nullable=False),
         sa.Column("name", sa.Text(), nullable=False),
+        # TODO: We should move type column to settings as an attribute (as we are already doing for Fields and Questions).
         sa.Column("type", sa.Text(), nullable=False),
-        sa.Column("description", sa.Text(), nullable=True),
+        sa.Column("description", sa.Text()),
         sa.Column("settings", sa.JSON(), nullable=False),
+        sa.Column("allowed_roles", sa.JSON, server_default="[]"),
         sa.Column("dataset_id", sa.Uuid(), nullable=False),
-        sa.Column("id", sa.Uuid(), nullable=False),
         sa.Column("inserted_at", sa.DateTime(), nullable=False),
         sa.Column("updated_at", sa.DateTime(), nullable=False),
         sa.ForeignKeyConstraint(["dataset_id"], ["datasets.id"], ondelete="CASCADE"),
```

Comment:
we have to make sure that running `alembic -c src/argilla/alembic.ini revision --autogenerate -m "test"` generates an empty revision meaning that the metadata in the class models is aligned with the metadata in this migration file

Reply:
I will fix the migration on the new PR with the changes for updating metadata properties. Thanks for the comment.